### DISCUSSION TITLE
how can i view issues created by me

### DISCUSSION THREAD

**Discussion by sf8193** [ankitpokhrel/jira-cli#598] [Q&A] [answered]
I've tried --created but that's for time, anyway i can see issues created by me? Also Are there any more in depth docs than the readme? Can't seem to find them

**Comment by sf8193** [comment, marked as answer]
found it, seems to be this `jira issue list -r $(jira me)` although would really like to know if/where there are more in depth docs than trying to grope around cli help command.

Very exciting tool, don't want to take away from that! I might actually enjoy using jira now

**Reply by ankitpokhrel** [reply]
Hi @sf8193, thank you for the feedback and glad to know that you are enjoying the tool. Readme and help are the only documentation at the moment, I will try to improve the docs wherever I can. Is there any particular section/category/example that you would like to have in the docs?

**Reply by sf8193** [reply]
in the docs i think it says sprint flags are the same as issue flags, but that's not represented in help and they don't _seem_ to behave exactly the same? Or i'm missing something
